test/python/test_ibmqjob.py

```diff
@@ -282,6 +283,46 @@ def test_retrieve_job_error(self):
         backend = lowest_pending_jobs(backends)
         self.assertRaises(IBMQBackendError, backend.retrieve_job, 'BAD_JOB_ID')
 
+    def test_get_jobs_filter_job_status(self):
+        backends = self._provider.available_backends()
+        backend = lowest_pending_jobs(backends)
+        job_list = backend.jobs(limit=5, skip=0, status=JobStatus.DONE)
+        self.log.info('found %s matching jobs', len(job_list))
+        for i, job in enumerate(job_list):
+            self.log.info('match #%d: %s', i, job.result()._result['status'])
+            self.assertTrue(job.status['status'] == JobStatus.DONE)
+
+    def test_get_jobs_filter_counts(self):
```

Comment:
What about first sending the jobs before testing them? I'm only worried about what happens if there's no such jobs in the server, so the test always passes and is not testing anything. (... maybe am I being to much skeptical? :))

Reply:
I agree. I think I did this because of the trouble with assuring a job status for testing. It's a good argument for mocking the server.